doc/modules/linear_model.rst

```diff
@@ -111,6 +114,33 @@ its ``coef_`` member::
     0.13636...
 
 
+Classification
+--------------
+
+The :class:`Ridge` regressor has a classifier variant:
+:class:`RidgeClassifier`. This classifier first converts binary targets to
+``{-1, 1}`` and then treats the problem as a regression task, optimizing
+the same objective as above. The predicted class corresponds to the sign of
+the regressor's prediction. For multiclass classification, the problem is
+treated as multi-output regression.
```

Comment:
and the predicted class corresponds to ...

Reply:
the output with the highest predicted value.